client/package.json

```diff
@@ -4,22 +4,22 @@
 	"private": true,
 	"proxy": "http://localhost:3001/",
 	"dependencies": {
+		"@apollo/client": "^3.3.16",
 		"@fortawesome/fontawesome-svg-core": "^1.3.0",
 		"@fortawesome/free-brands-svg-icons": "^6.0.0",
 		"@fortawesome/free-solid-svg-icons": "^6.0.0",
 		"@fortawesome/react-fontawesome": "^0.1.17",
 		"@testing-library/jest-dom": "^5.16.2",
 		"@testing-library/react": "^12.1.4",
 		"@testing-library/user-event": "^13.5.0",
+		"graphql": "^15.5.0",
+		"jwt-decode": "^3.1.2",
 		"react": "^17.0.2",
 		"react-dom": "^17.0.2",
 		"react-router-dom": "^6.2.2",
 		"react-scripts": "5.0.0",
 		"sass": "^1.49.9",
-		"web-vitals": "^2.1.4",
-		"@apollo/client": "^3.3.16",
```

Comment:
Not sure what you done here. You are reordering the dependencies in the file and you removed the apollo/client. Are you sure you did a git pull origin develop before pushing your commit?

Reply:
At some point I did npm install, that probably rearranged things. @apollo/client went up, it is still there